README.md

```diff
@@ -21,19 +21,34 @@ git clone https://github.com/luizdepra/hugo-coder.git themes/coder
 Add the following lines to your `config.toml`.
 
 ```toml
-theme = "coder" # set the theme
```

Comment:
I would suggest to keep the comment in README file as they are descriptive.

Reply:
Other than that it LGTM.

Reply:
Agreed.